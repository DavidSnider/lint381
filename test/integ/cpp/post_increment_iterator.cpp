for (auto it = foo.begin(); it != foo.end(); it++) {

}

for (auto it = foo.begin(); it != foo.end(); ++it) {

}

if ((it)) {

}

while (*it++ = *it++) {

}
