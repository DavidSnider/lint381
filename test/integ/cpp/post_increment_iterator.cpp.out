post_increment_iterator.cpp:1:46: error: Use pre-increment instead of post-increment for iterators in loops
for (auto it = foo.begin(); it != foo.end(); it++) {
                                             ^^^^
